blocks/library/quote/index.js

```diff
@@ -20,11 +20,21 @@ registerBlock( 'core/quote', {
 		citation: html( 'footer' )
 	},
 
+	controls: [ '1', '2' ].map( ( level ) => ( {
```

Comment:
Minor: Do you think it's any different or better to treat `'1'` as a string vs. `1` as a number?